src/program.js

```diff
@@ -19,15 +19,25 @@ export class Program {
   commands: { [key: string]: Function };
   shouldExitProgram: boolean;
 
-  constructor(argv: ?Array<string>, {yargsInstance=yargs}: Object = {}) {
-    if (argv !== undefined) {
-      // This allows us to override the process argv which is useful for
-      // testing.
-      yargsInstance = yargs(argv);
-    }
+  constructor(
+    argv: ?Array<string>,
+    {absolutePackageDir=process.cwd()}: {absolutePackageDir?: string} = {}
+  ) {
+    // This allows us to override the process argv which is useful for
+    // testing.
+    // NOTE: process.argv.slice(2) removes the path to node and web-ext
+    // executables from the process.argv array.
+    argv = argv || process.argv.slice(2);
```

Comment:
could we just leave this as `undefined` when not overridden? That should trigger the `yargs` default behavior.